core/bindings.rs

```diff
@@ -438,7 +464,41 @@ fn opcall_async<'s>(
   }
 }
 
-fn set_macrotask_callback(
+fn has_tick_scheduled(
+  scope: &mut v8::HandleScope,
+  _args: v8::FunctionCallbackArguments,
+  mut rv: v8::ReturnValue,
+) {
+  let state_rc = JsRuntime::state(scope);
+  let state = state_rc.borrow();
+  rv.set(to_v8(scope, state.has_tick_scheduled).unwrap());
+}
+
+fn set_has_tick_scheduled(
+  scope: &mut v8::HandleScope,
+  args: v8::FunctionCallbackArguments,
+  _rv: v8::ReturnValue,
+) {
+  let state_rc = JsRuntime::state(scope);
+  let mut state = state_rc.borrow_mut();
+
+  let value = match v8::Local::<v8::Boolean>::try_from(args.get(0)) {
+    Ok(cb) => cb,
+    Err(err) => return throw_type_error(scope, err.to_string()),
+  };
+
+  state.has_tick_scheduled = value.is_true();
```

Comment:
```suggestion
  state.has_tick_scheduled = args.get(0).is_true();
```
Or is there a reason to do the conversion first?

Reply:
Sure, that works